torchrl/objectives/a2c.py

```diff
@@ -388,7 +396,7 @@ def get_entropy_bonus(self, dist: d.Distribution) -> torch.Tensor:
             entropy = dist.entropy()
         except NotImplementedError:
             x = dist.rsample((self.samples_mc_entropy,))
-            entropy = -dist.log_prob(x)
+            entropy = -dist.log_prob(x).mean(0)
```

Comment:
why 0? What if the input tensordict has 2 or 3 dims? Don't we want to mean across the number of dims of the input tensordict?

Reply:
Because dist.rsample generates `self.samples_mc_entropy` samples along dimension 0. So we average then also in dim 0 to get an estimation of the entropy for each position of the input tensordict.

Reply:
Oh right!